tests/sample_eventgen_conf/unit/eventgen.conf.config

```diff
@@ -0,0 +1,38 @@
+[sample]
+sampleDir = tests/sample_eventgen_conf/sample
+outputMode = splunkstream
+splunkMethod = https
+splunkHost = localhost
+splunkPort = 8088
+earliest = -3s
+latest = now
+
+count = 0
```

Comment:
To use the full sample, we need to use count = -1.

Reply:
changed.